src/main/java/de/picturesafe/search/samples/facet/Facet.java

```diff
@@ -20,8 +36,14 @@
 
 import java.util.stream.LongStream;
 
+/**
+ * Aggregation/Facet sample.
+ *
+ * @deprecated Use samples of package {@link de.picturesafe.search.samples.aggregations} instead.
```

Comment:
Just for curiosity: why are we keeping this sample?

Reply:
It is still referenced in https://picturesafe-search.io/docs/reference/aggregations/.
After modification the sample can be removed.